dvc/repo/ls.py

```diff
@@ -40,13 +40,13 @@ def ls(
             ret = _ls(repo, path_info, recursive, True)
 
         nondvc = {}
-        if not outs_only:
+        if not dvc_only:
             nondvc = _ls(repo, path_info, recursive, False)
 
         ret.update(nondvc)
 
         if path and not ret:
-            raise PathMissingError(path, repo, output_only=outs_only)
+            raise PathMissingError(path, repo, output_only=dvc_only)
```

Comment:
Shouldn't we change the param name here too? It seems to me that now it will be a little detached from rest of the code that uses `dvc_only`. I guess its a matter of opinion, so no pressure.

Reply:
I think it definitely makes sense to change it here as well, since `dvc ls` is the only place we even use the `output_only` parameter for `PathMissingError`